app/components/submission/header.js

```diff
@@ -381,6 +391,7 @@ export default class SubmissionHeaderComponent extends Component {
   );
 
   takeInTreatment = async () => {
+    // TODO update submission data? It could have been changed on subcase
```

Comment:
out of scope (some of this code came from ACC flags.
When BIS submission is ongoing, any updates to the subcase do not cascade to the submission.
If we were to update the submission, this might be the place to do it (before editors start treating)
They could then auto update and send back if needed 'something changed on subcase, change ... on your submission'. I don't know